### PR TITLE
docs(general): update with OpenClarity

### PR DESCRIPTION
**Pull request by akijakya**
## Description

Update logos, gif, and string from VMClarity to their OpenClarity counterparts, where it was possible.
Left comments where there are some prerequisites before the change.

## Type of Change

- [ ] Bug Fix
- [ ] New Feature
- [ ] Breaking Change
- [ ] Refactor
- [X] Documentation
- [ ] Other (please describe)

## Checklist

- [X] I have read the [contributing guidelines](https://github.com/openclarity/openclarity/blob/main/CONTRIBUTING.md)
- [ ] Existing issues have been referenced (where applicable)
- [X] I have verified this change is not present in other open pull requests
- [ ] Functionality is documented
- [X] All code style checks pass
- [ ] New code contribution is covered by automated tests
- [X] All new and existing tests pass

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,31 @@
...
-  <source media="(prefers-color-scheme: dark)" srcset="./assets/logos/VMClarity-logo-dark-bg-horizontal@4x.png">
-  <source media="(prefers-color-scheme: light)" srcset="./assets/logos/VMClarity-logo-light-bg-horizontal@4x.png">
-  <img alt="VMClarity Logo" src="./assets/logos/VMClarity-logo-light-bg-horizontal@4x.png">
+  <source media="(prefers-color-scheme: dark)" srcset="./assets/logos/OpenClarity-logo-dark-bg.png">
+  <source media="(prefers-color-scheme: light)" srcset="./assets/logos/OpenClarity-logo-light-bg.png">
+  <img alt="OpenClarity Logo" src="./assets/logos/OpenClarity-logo-light-bg.png">
 </picture>
 
 [![Slack Invite](https://img.shields.io/badge/Slack-Join-blue?logo=slack)](https://outshift.slack.com/messages/vmclarity)
-[![Go Reference](https://pkg.go.dev/badge/github.com/openclarity/vmclarity.svg)](https://pkg.go.dev/github.com/openclarity/vmclarity)
```
Comment (akijakya) [inline README.md:8]: Tried adding the project to `pkg.go.dev` but it didn't succeed

File: README.md [hunk 2]
```diff
@@ -1,28 +1,31 @@
...
-![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/openclarity/vmclarity/main-merge.yml?style=flat-square&branch=main)
-[![Go Report Card](https://goreportcard.com/badge/github.com/openclarity/vmclarity)](https://goreportcard.com/report/github.com/openclarity/vmclarity)
 
-VMClarity is an open source tool for agentless detection and management of Virtual Machine
+<!--[![Go Reference](https://pkg.go.dev/badge/github.com/openclarity/vmclarity.svg)](https://pkg.go.dev/github.com/openclarity/vmclarity)-->
```
Comment (akijakya) [inline README.md:9]: Tried adding the project to pkg.go.dev but it didn't succeed

File: README.md [hunk 3]
```diff
@@ -1,28 +1,31 @@
...
+
+![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/openclarity/openclarity/main-merge.yml?style=flat-square&branch=main)
+
+<!--[![Go Report Card](https://goreportcard.com/badge/github.com/openclarity/vmclarity)](https://goreportcard.com/report/github.com/openclarity/vmclarity)-->
```
Comment (akijakya) [inline README.md:13]: Tried adding the project on the Go report card website but it didn't succeed (I think the problem is the project being multi-module)...

File: README.md [hunk 4]
```diff
@@ -1,28 +1,31 @@
...
+
+OpenClarity is an open source tool for agentless detection and management of Virtual Machine
 Software Bill Of Materials (SBOM) and security threats such as vulnerabilities, exploits, malware, rootkits, misconfigurations and leaked secrets.
 
-<img src="./assets/vmclarity_demo.gif" alt="VMClarity demo" />
+<img src="./assets/OpenClarity-demo.gif" alt="OpenClarity demo" />
 
-Join [VMClarity's Slack channel](https://outshift.slack.com/messages/vmclarity) to hear about the latest announcements and upcoming activities. We would love to get your feedback!
+Join [OpenClarity's Slack channel](https://outshift.slack.com/messages/vmclarity) to hear about the latest announcements and upcoming activities. We would love to get your feedback!
```
Comment (akijakya) [inline README.md:20]: The slack channel should be renamed to `openclarity` or a new channel should be created?

File: README.md [hunk 5]
```diff
@@ -1,28 +1,31 @@
 <picture>
-  <source media="(prefers-color-scheme: dark)" srcset="./assets/logos/VMClarity-logo-dark-bg-horizontal@4x.png">
-  <source media="(prefers-color-scheme: light)" srcset="./assets/logos/VMClarity-logo-light-bg-horizontal@4x.png">
-  <img alt="VMClarity Logo" src="./assets/logos/VMClarity-logo-light-bg-horizontal@4x.png">
+  <source media="(prefers-color-scheme: dark)" srcset="./assets/logos/OpenClarity-logo-dark-bg.png">
+  <source media="(prefers-color-scheme: light)" srcset="./assets/logos/OpenClarity-logo-light-bg.png">
+  <img alt="OpenClarity Logo" src="./assets/logos/OpenClarity-logo-light-bg.png">
 </picture>
 
 [![Slack Invite](https://img.shields.io/badge/Slack-Join-blue?logo=slack)](https://outshift.slack.com/messages/vmclarity)
```
Comment (akijakya) [inline README.md:7]: The slack channel should be renamed to `openclarity` or a new channel should be created?

File: README.md (hunk 3 above)
Comment (ramizpolic) [inline README.md:13]: nit: minor cosmetic updates to the README header

```md
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="./assets/logos/OpenClarity-logo-dark-bg.png">
  <source media="(prefers-color-scheme: light)" srcset="./assets/logos/OpenClarity-logo-light-bg.png">
  <img alt="OpenClarity Logo" src="./assets/logos/OpenClarity-logo-light-bg.png">
  <br/><br/><br/>
</picture>

[![Slack Invite](https://img.shields.io/badge/Slack-Join-blue?logo=slack)](https://outshift.slack.com/messages/vmclarity)
[![Go Reference](https://pkg.go.dev/badge/github.com/openclarity/openclarity.svg)](https://pkg.go.dev/github.com/openclarity/openclarity)
![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/openclarity/openclarity/main-merge.yml?style=flat-square&branch=main)
[![Go Report Card](https://goreportcard.com/badge/github.com/openclarity/openclarity)](https://goreportcard.com/report/github.com/openclarity/openclarity)
```

File: README.md [hunk 6]
```diff
@@ -73,15 +79,15 @@ VMClarity uses a pluggable scanning infrastructure to provide:
...
+enabled/disabled on an individual basis. OpenClarity normalizes, merges and
 provides a robust visualization of the results from these various tools.
 
 These tools include:
 
 - SBOM Generation and Analysis
   - [Syft](https://github.com/anchore/syft)
   - [Trivy](https://github.com/aquasecurity/trivy)
-  - [Windows Registry](cli/analyzer/windows)*
+  - [Windows Registry](cli/analyzer/windows)\*
```
Comment (ramizpolic) [inline README.md:90]: reference update
```suggestion
  - [Windows Registry](scanner/families/sbom/windows)\*
```

File: README.md (hunk 2 above)
Comment (ramizpolic) [reply README.md:9]: note that this will happen automatically after we tag our first OpenClarity version

File: README.md (hunk 3 above)
Comment (ramizpolic) [reply README.md:13]: this also should be resolved after tagging/release

File: README.md [hunk 7]
```diff
@@ -73,15 +79,15 @@ VMClarity uses a pluggable scanning infrastructure to provide:
...
-enabled/disabled on an individual basis. VMClarity normalizes, merges and
+enabled/disabled on an individual basis. OpenClarity normalizes, merges and
 provides a robust visualization of the results from these various tools.
 
 These tools include:
 
 - SBOM Generation and Analysis
   - [Syft](https://github.com/anchore/syft)
   - [Trivy](https://github.com/aquasecurity/trivy)
-  - [Windows Registry](cli/analyzer/windows)*
```
Comment (ramizpolic) [inline README.md:84]: btw widnows registry scanner is supporetd for all hosts (windows, unix, mac)

File: docs/test_e2e.md [hunk 8]
```diff
@@ -2,39 +2,39 @@
...
+### 3. Ensure that OpenClarity backend is working correctly
 
 1. Get the IP address from the CloudFormation stack's Output Tab
 2. `ssh ubuntu@<ip address>`
-3. Check the VMClarity Logs
+3. Check the OpenClarity Logs
 
    ```shell
-   sudo journalctl -u vmclarity
+   sudo journalctl -u openclarity
```
Comment (paralta) [inline docs/test_e2e.md:37]: The service name is still `vmclarity` here, this is one of the missing renames along with logs and filepaths https://github.com/openclarity/openclarity/pull/756#issue-2463390832. But I do not believe this is a blocker, I can update this after this PR is merged

**Review by paralta — APPROVED** [review]
looking good 🚀

File: README.md (hunk 7 above)
Comment (akijakya) [reply README.md:84]: Should I add something here?

File: docs/test_e2e.md (hunk 8 above)
Comment (akijakya) [reply docs/test_e2e.md:37]: Thanks for noticing!

**Review by ramizpolic — APPROVED** [review]

File: README.md (hunk 7 above)
Comment (ramizpolic) [reply README.md:84]: removing the `*` and `* Windows only` from the readme should be enough

**Comment by ramizpolic** [comment]
Really nice dashboard preview, thansk @akijakya

**Review by lgecse — APPROVED** [review]

**Pull request merged**